### ISSUE TITLE
Feature request: expose and allow individual messages to be starred

### ISSUE THREAD

**Issue by twpol** [nylas/N1#1749]
- I'm running Nylas N1 0.4.14 on Windows 10 (version 1511, build 14279.1000).
- I have the Emoji Keyboard, Link Tracking, Open Tracking, Send Later plug-ins enabled (the defaults I think).
- I'm using the IMAP setup with Outlook.com (because the Outlook.com option didn't work).

I would like to star and see stars on individual messages.

The current behaviour with stars is bizarre and confusing if any other client is used to look at the messages and is not subtle enough for me: adding/removing a star adds/removes the flag marker from ALL messages in that conversation.

Similar to #1401, I use stars to track actionable messages and those which need a reply, so it is crucial that:
- When a message *within* a conversation is starred, I can see which one it is (I cannot currently).
- When I set a message for later action, I can set it for just that message (I cannot currently).

This behaviour basically makes it impossible for me to use Nylas N1, as reading messages will necessarily need some of those messages to be individually flagged.

**Comment by luetge** [comment]
I completely agree, this makes the client useless for me, too. I bet there are tons of people using message starring to organize their todos and probably almost none of them will change their habits just to use this otherwise awesome app, too bad.
